google/cloud/storage/_opentelemetry_tracing.py

```diff
@@ -0,0 +1,101 @@
+"""Manages OpenTelemetry tracing span creation and handling."""
+
+import logging
+import os
+
+from contextlib import contextmanager
+
+from google.api_core import exceptions as api_exceptions
+from google.api_core import retry as api_retry
+from google.cloud.storage import __version__
+from google.cloud.storage.retry import ConditionalRetryPolicy
+
+
+ENABLE_OTEL_TRACES_ENV_VAR = "ENABLE_GCS_PYTHON_CLIENT_OTEL_TRACES"
+_DEFAULT_ENABLE_OTEL_TRACES_VALUE = False
+
+enable_otel_traces = os.environ.get(
+    ENABLE_OTEL_TRACES_ENV_VAR, _DEFAULT_ENABLE_OTEL_TRACES_VALUE
+)
+logger = logging.getLogger(__name__)
+
+try:
+    from opentelemetry import trace
+
+    HAS_OPENTELEMETRY = True
+
+    logger.info(f"HAS_OPENTELEMETRY is {HAS_OPENTELEMETRY}")
```

Comment:
This looks a little terse, is it intended to be user-facing? Seems like we could put it outside of the try/except if so.

Reply:
Yep only the logger.debug message is needed; Removed this line :)